pages/schedule.tsx

```diff
@@ -165,6 +166,8 @@ export default function Schedule({ filters, shifts }: ISchedulesProps) {
           />
         </div>
 
+        <Label />
+
```

Comment:
Why is it necessary to create a component just for that link? 
Creating a component is just needed when something is used more then once or when some component logic is to big. I think you can just put the link directly in this page.